packages/arb-token-bridge-ui/tests/support/common.ts

```diff
@@ -183,6 +183,28 @@ export const startWebApp = (url = '/', qs: { [s: string]: string } = {}) => {
   })
 }
 
+export async function getNativeTokenDecimals({
```

Comment:
This is currently inside the test files, would it be useful to move inside the codebase?